### PR TITLE
[116] Update to TensorFlow 1.3.0

### PR DESCRIPTION
**Pull request by thunterdb**
Note: this PR builds on https://github.com/databricks/tensorframes/pull/120

### CODE REVIEW

File: src/main/python/tensorframes/core_test.py [hunk 1]
```diff
@@ -91,6 +91,20 @@ def test_map_blocks_3(self):
         data2 = df2
         assert data2.z[0] == 3.0, data2
 
+    def test_map_blocks_feed_dict(self):
+        data = [dict(x_spark=float(x)) for x in range(10)]
+        df = pd.DataFrame(data)
+        with tf.Graph().as_default() as g:
```
Comment (phi-dbq) [inline src/main/python/tensorframes/core_test.py]: We probably don't need the `as g` if we are not directly using `g`.

File: src/main/python/tensorframes/core_test.py [hunk 2]
```diff
@@ -130,6 +144,19 @@ def test_map_rows_3(self):
         data2 = df2
         assert data2.z[0] == 3.0, data2
 
+    def test_map_rows_feed_dict(self):
+        data = [dict(x_spark=float(x)) for x in range(5)]
+        df = pd.DataFrame(data)
+        with tf.Graph().as_default() as g:
+            # The placeholder that corresponds to column 'x'
+            x = tf.placeholder(tf.double, shape=[None], name="x_tf")
```
Comment (phi-dbq) [inline src/main/python/tensorframes/core_test.py]: Will it work if the dtype of `x` is set to `tf.float32`?

File: src/main/python/tensorframes/core_test.py [hunk 3]
```diff
@@ -91,6 +91,20 @@ def test_map_blocks_3(self):
         data2 = df2
         assert data2.z[0] == 3.0, data2
 
+    def test_map_blocks_feed_dict(self):
```
Comment (phi-dbq) [inline src/main/python/tensorframes/core_test.py]: Shall we also add a test with spark dataframes?

Comment (thunterdb) [inline src/main/python/tensorframes/core_test.py]: My mistake, I meant to check against spark dataframes.

File: src/main/python/tensorframes/core_test.py (hunk 1 above)
Comment (thunterdb) [inline src/main/python/tensorframes/core_test.py]: indeed, thanks.

**Comment by phi-dbq** [comment]
LGTM!

**Comment by thunterdb** [comment]
Thanks @phi-dbq , merging.

**Pull request merged**